lua/tabline.lua

```diff
@@ -42,11 +43,18 @@ local function tabline(options)
         end
         -- buf name
         s = s .. options.brackets[1]
+        local pre_title_s_len = string.len(s)
         if bufname ~= '' then
             s = s .. icon .. fn.fnamemodify(bufname, ':t')
         else
             s = s .. options.no_name
         end
+        if options.inactive_tab_max_length
+            and options.inactive_tab_max_length ~= 0
```

Comment:
Actually, this condition could be set to something like `options.inactive_tab_max_length > 0`. Please let me know if that's what you prefer (to be on a safe side).

Reply:
> Actually, this condition could be set to something like `options.inactive_tab_max_length > 0`. Please let me know if that's what you prefer (to be on a safe side).

This sounds good to me.

Reply:
Thanks. Update coming up..